### PR TITLE
Error bot

### PR DESCRIPTION
**Pull request by ashryanbeats**

### CODE REVIEW

**Comment by ashryanbeats** [comment]
Note that you'll need a env var: `SLACK_ALERT_CHANNEL_ID`

File: src/slack/error.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+const channel = config.SLACK_ALERT_CHANNEL_ID;
+
+let prevErrorMessage = "";
+let prevStackTrace = "";
+let threadTs = "";
+
+// For use by SlackErrorTranport in winston
+const sendAlert = async (error) => {
+  // For new errors (i.e., not the same error as the previous one)
```
Comment (bcosynot) [inline src/slack/error.js:13]: Can you make it exit the function if `SLACK_ALERT_CHANNEL_ID` is not provided?

Comment (ashryanbeats) [reply src/slack/error.js:13]: Yep! Will revise today.

Comment (ashryanbeats) [reply src/slack/error.js:16]: This refinement is included in the commits below. Lmk if it's what you had in mind. Thanks for bringing this up. Something was bugging me yesterday and I couldn't quite see it, but this was it.

Comment (bcosynot) [reply src/slack/error.js:16]: Yep, looks good. Thanks for the update!

**Pull request merged**